backend/db/migrate/20170801101708_create_clients.rb

```diff
@@ -0,0 +1,9 @@
+class CreateClients < ActiveRecord::Migration[5.1]
+  def change
+    create_table :clients do |t|
+      t.string :token
+
```

Comment:
Please add a migration file for this. Reference: http://guides.rubyonrails.org/active_record_migrations.html